src/WebJobs.Script/Description/DotNet/DotNetConstants.cs

```diff
@@ -17,5 +18,162 @@ public static class DotNetConstants
         public const string InvalidFileMetadataReferenceCode = "AF006";
         public const string InvalidEntryPointNameCompilationCode = "AF007";
         public const string AsyncVoidCode = "AF008";
+
+        public static string[] FrameworkReferences = new[]
```

Comment:
>FrameworkReferences [](start = 31, length = 19)

Why do we need this list?

Reply:
These references are needed when targeting .NET Standard 2.0 during compilation.